test/Makefile

```diff
@@ -1,9 +1,9 @@
 .PHONY: default pr all
 
 default: check-deps test 
-pr: check-deps test-pr
-# TODO: should have a separate target
-all: check-deps test-pr ltp
+platforms: check-deps test-platforms
+pr: default
```

Comment:
what's the difference between `default` and `pr`. Can one of them be removed, or a more common pattern would be to have `default: pr` and then `pr: ...`